packages/cli/src/commands/bundle.ts

```diff
@@ -156,6 +156,13 @@ export async function handleBundle(
           `📦 Created a bundle for ${blue(entrypoint)} at ${blue(outputFile)} ${green(elapsed)}.\n`,
         );
       }
+
+      const removedCount = meta.visitorsData['remove-unused-components']?.removedCount;
+      if (removedCount) {
+        process.stderr.write(
```

Comment:
why is this stderr and not stdout?

Reply:
Bundle goes into sdtout (if -o is not provided). All the logs go into stderr. This is common for CLI tools.